examples/jsm/postprocessing/UnrealBloomPass.js

```diff
@@ -298,6 +298,16 @@ class UnrealBloomPass extends Pass {
 
 	getSeperableBlurMaterial( kernelRadius ) {
 
+		const coefficients = [];
+
+		const sigma = kernelRadius;
+
+		for ( let i = 0; i < kernelRadius; i ++ ) {
+
+			coefficients.push( 0.39894 * Math.exp( - 0.5 * i * i / ( sigma * sigma ) ) / sigma );
```

Comment:
Can we just write here all `kernelRadius` instead of `sigma`?

Also, I think the define `'SIGMA': kernelRadius` later is unused now and can be removed.

Reply:
Thank you for pointing out to these! Cleaned these up.